app/build.gradle

```diff
@@ -109,6 +110,20 @@ dependencies {
 	androidTestImplementation 'androidx.test:runner:1.2.0'
 	androidTestImplementation 'androidx.test:rules:1.2.0'
 	implementation project(path: ':engine')
+
+	androidTestImplementation 'tools.fastlane:screengrab:2.0.0'
+	androidTestImplementation 'androidx.test:runner:1.2.0'
+	androidTestImplementation 'androidx.test:rules:1.2.0'
+	androidTestImplementation ('androidx.test.espresso:espresso-contrib:3.2.0'){
+		exclude group: 'com.android.support', module: 'appcompat'
+		exclude group: 'com.android.support', module: 'support-v4'
+		exclude module: 'recyclerview-v7'
+	}
+	androidTestImplementation ('androidx.test.espresso:espresso-core:3.2.0'){
+		exclude group: 'com.android.support', module: 'appcompat'
+		exclude group: 'com.android.support', module: 'support-v4'
+		exclude module: 'recyclerview-v7'
+	}
```

Comment:
What is the reason why we're excluding these groups?

Reply:
it's how it's in the documentation